### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by XsidX**
For this PR, I:
  - [install linter files, set up local linters](https://github.com/XsidX/oop_school_library/commit/99f8fdeb2e26b1d088ed94a068567a01d40781b3)
  - [create person class](https://github.com/XsidX/oop_school_library/commit/6ed58c86a30fefe8d51b1d97dd7734813ae0d3c9)
  - [create student class](https://github.com/XsidX/oop_school_library/commit/3f767a9a6a551fd2a016496412270830f0afb2a1)
  - [create teacher class](https://github.com/XsidX/oop_school_library/commit/706dae23ad20181f105094c9a91fd3ad9f3f9463)

### CODE REVIEW

File: student.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Student < Person
+  def initialize(classroom)
+    super(age)
+    @classroom = classroom
+  end
```
Comment (mikethreels) [inline student.rb:7]: - [ ] The constructor should extend the parent's constructor by adding @specialization and a parameter for it.  have a look at this Link to the example about constructors in a previous lesson https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/class_and_object_for_irb.md#constructor-and-instance-vars

File: teacher.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+require_relative 'student'
+
+class Teacher < Person
+  def initialize(specialization)
+    super(age)
+    @specialization = specialization
+  end
```
Comment (mikethreels) [inline teacher.rb:7]: - [ ] The constructor should extend the parent's constructor by adding @specialization and a parameter for it.  have a look at this Link to the example about constructors in a previous lesson https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/class_and_object_for_irb.md#constructor-and-instance-vars

**Review by mikethreels — APPROVED** [review]
Hi Sid, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- [ ] linters are passing
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: student.rb (hunk 1 above)
Comment (XsidX) [reply student.rb:7]: Hi, @mikethreels 
Thanks for the feedback, but I think the specialization instance variable belongs to the Teacher class, not Student class.

Comment (mikethreels) [reply student.rb:7]: Hi, @XsidX, you are right I forgot to change the name of the instance variable after copy and pasting it.

**Pull request merged**